Cargo.lock

```diff
@@ -36,7 +36,7 @@ dependencies = [
  "tempdir 0.3.4 (registry+https://github.com/rust-lang/crates.io-index)",
  "threadpool 1.3.0 (registry+https://github.com/rust-lang/crates.io-index)",
  "time 0.1.35 (registry+https://github.com/rust-lang/crates.io-index)",
- "tipb 0.0.1 (git+https://github.com/pingcap/tipb.git)",
+ "tipb 0.0.1 (git+https://github.com/pingcap/tipb.git?branch=hhkbp2/add-expr-data-types)",
```

Comment:
Could you change `tipb` since the related PR is already merged and  the branch `hhkbp2/add-expr-data-types` was removed